cmd/view-fits/main.go

```diff
@@ -119,39 +138,64 @@ func main() {
 					return
 				case key.CodeRightArrow:
 					if e.Direction == key.DirPress {
-						if index < nbImages-1 {
-							index++
+						if ifile < nbFiles-1 {
+							ifile++
 						} else {
-							index = 0
+							ifile = 0
 						}
+						iimg = 0
 						b.Release()
 						w.Send(paint.Event{})
 					}
 				case key.CodeLeftArrow:
 					if e.Direction == key.DirPress {
-						if index == 0 {
-							index = nbImages - 1
+						if ifile == 0 {
+							ifile = nbFiles - 1
 						} else {
-							index--
+							ifile--
 						}
+						iimg = 0
 						b.Release()
 						w.Send(paint.Event{})
 					}
+
+				case key.CodeUpArrow:
```

Comment:
according to the doc, `UpArrow` is the `previous` image.
I think, at least for western users, that makes sense.

could you please switch `key.CodeUpArrow` with `key.DownArrow` below ?